### PR TITLE
Update foshttpcachebundle

### PR DESCRIPTION
**Pull request by itkg-ngilain**
https://github.com/open-orchestra/open-orchestra-display-bundle/pull/163
https://github.com/open-orchestra/open-orchestra-front-bundle/pull/114

### CODE REVIEW

File: DisplayBundle/Manager/CacheableManager.php [hunk 1]
```diff
@@ -27,13 +21,9 @@ class CacheableManager
      * @param CacheManager $cacheManager
```
Comment (itkg-alavieille) [inline DisplayBundle/Manager/CacheableManager.php]: remove this comment

File: DisplayBundle/Manager/CacheableManager.php [hunk 2]
```diff
@@ -88,14 +78,13 @@ protected function setResponseMaxAge(Response $response, $maxAge, $status)
...
     /**
-     * Tag response
+     * Add tags to include in the response
      * 
-     * @param Response $response
-     * @param array    $tags
+     * @param array $tags
      */
-    public function tagResponse(Response $response, $tags)
+    public function addCacheTags($tags)
```
Comment (itkg-alavieille) [inline DisplayBundle/Manager/CacheableManager.php]: you can add array in the method signature  ```public function addCacheTags(array $tags)```

File: DisplayBundle/Tests/Manager/CacheableManagerTest.php [hunk 3]
```diff
@@ -77,15 +75,13 @@ public function provideMaxAge()
 
     /**
      */
```
Comment (itkg-alavieille) [inline DisplayBundle/Tests/Manager/CacheableManagerTest.php]: add a method comment

**Comment by itkg-ngilain** [comment]
@itkg-alavieille : update done

**Comment by itkg-alavieille** [comment]
:+1:

**Pull request merged**